### PR TITLE
Fix Inchi being hardcoded into PostgreSQL

### PR DESCRIPTION
**Pull request by ptosco**
This PR should fix the problem with "Inchi" being hardcoded into PostgreSQL.
@giallu It would be great if you could test it in your context and check that it works.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -127,6 +127,47 @@ if(MSVC)
   ADD_DEFINITIONS("-DBOOST_ALL_NO_LIB")
 endif(MSVC)
 
+if(RDK_BUILD_INCHI_SUPPORT)
```
Comment (greglandrum) [inline CMakeLists.txt]: I'm not thrilled with having this in CMakeLists.txt
If it needs to be separate (and it looks like it does), I would prefer to have this done in $RDBASE/Code/cmake/modules/FindInchi.cmake

Comment (ptosco) [inline CMakeLists.txt]: Yeah, I agree. Anyway it needs more testing: it is failing on Travis. WIll give it a go as soon as I have a bit of time.

**Comment by greglandrum** [comment]
let me know if you want me to take a look.

**Comment by ptosco** [comment]
Not sure why AppVeyor failed?

**Comment by giallu** [comment]
WORKSFORME. Thanks a lot Paolo

**Comment by greglandrum** [comment]
Thanks Paolo.

**Pull request merged**